ResultsData.cs

```diff
@@ -105,5 +105,21 @@ internal class ResultsData
     {
         public Dictionary<string, FlavorData> results = new();
     }
+
+    internal class GraphPointData
+    {
+        public string dateTime;
+        public string taskname;
```

Comment:
this should be named like taskMeasurementName as the string you are passing here contains both. for example `AppStart` is the task name and `Page Show` is the measurement name here.